dist/pmAnimatedBanners.js

```diff
@@ -4,6 +4,78 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
 });
+
+var _util = require('./util');
+
+var _util2 = _interopRequireDefault(_util);
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+/**
+ * Cursor module for controlling hover states in the canvas, will inject
+ * custom css rules into the DOM and return methods for changing the
+ * cursor.
+ *
+ * @return {Object} Object literal with methods for setting and getting
+ *                  the cursor, accepted cursor types 'pointers' and
+ *                  'default'
+ */
+
+exports.default = function () {
+  // inject css rules into the dom
+  (function (head) {
+    // set css rules
+    var css = '\n      canvas#canvas {\n        cursor: default !important;\n      }\n      canvas#canvas.has-pointer {\n        cursor: pointer !important;\n      }\n    ';
+    var style = document.createElement('style');
+
+    // insert css rules to stylesheet
+    style.type = 'text/css';
+    if (style.styleSheet) {
+      style.styleSheet.cssText = css;
+    } else {
+      style.appendChild(document.createTextNode(css));
+    }
+
+    // inject to head
+    head.appendChild(style);
+  })(document.head || document.getElementsByTagName('head')[0]);
+
+  var cursor = void 0;
+  return {
+
+    /**
+     * set cursor state for canvas
+     *
+     * @param  {String} state [the cursor state, accepts either 'pointer' or 'default']
+     * @return {Void} Void
```

Comment:
It always needs to have a return, even when its null (you just need to put Void) - airbnb eslint config throws an error if its not in there